lib/concerns/findable.rb

```diff
@@ -0,0 +1,11 @@
+module Concerns
+  module Findable
+    def find_by_name(name)
+      all.find { |findable| findable.name.casecmp(name).zero? }
```

Comment:
#### Code Quality - Nitpick
```findable``` isn't very descriptive of what ```all``` actually holds. ```model```might be a more descriptive name.